docs/template_qna.yaml

```diff
@@ -0,0 +1,52 @@
+version: 3
+domain: <The knowledge domain which is used in prompts to the teacher model during synthetic data generation. The domain should be brief such as the title to a textbook chapter or section.>
+created_by: <The GitHub username of the contributor.>
+seed_examples:
+  - context: |
+    <Context from the document associated with this set of sample q&a pairs.>
+    questions_and_answers:
+      - question: |
+        <A question used for synthetic data generation.>
```

Comment:
perhaps clarify "A question used for synthetic data generation based in previous context."

Reply:
This is copied from the schema. Since the question_and_answers are in the same map object as the context, I would see that the question_and_answers are about the adjacent context. :-)